ghedesigner/radial_numerical_borehole.py

```diff
@@ -260,31 +231,27 @@ def fill_radial_cell(self, radial_cell, resist_p_eq, resist_f_eq, resist_tg_eq):
         # load soil cells
         for idx in range(cell_summation, num_soil_cells + cell_summation):
             j = idx - cell_summation
-            cell_type = RadialCellType.SOIL
-            thickness = self.thickness_soil
-            inner_radius = self.r_borehole + j * thickness
-            center_radius = inner_radius + thickness / 2.0
-            outer_radius = inner_radius + thickness
+            inner_radius = self.r_borehole + j * self.thickness_soil
+            center_radius = inner_radius + self.thickness_soil / 2.0
+            outer_radius = inner_radius + self.thickness_soil
             conductivity = self.single_u_tube.soil.k
             rho_cp = self.single_u_tube.soil.rhoCp
             volume = pi * (outer_radius ** 2 - inner_radius ** 2)
             radial_cell[:, idx] = np.array(
                 [
-                    cell_type,
                     inner_radius,
                     center_radius,
                     outer_radius,
-                    thickness,
                     conductivity,
                     rho_cp,
                     self.init_temp,
                     volume,
                 ],
-                dtype=self.dtype,
+                dtype=np.double,
             )
         cell_summation += num_soil_cells
 
-    def calc_sts_g_functions(self, single_u_tube, final_time=None, calculate_at_bh_wall=False) -> tuple:
+    def calc_sts_g_functions(self, single_u_tube, final_time=None) -> tuple:
```

Comment:
No need to make room for this unused argument anymore. Both are computed upon each call.